apps/jetty.go

```diff
@@ -62,35 +61,30 @@ func init() {
 	confgenerator.MetricsReceiverTypes.RegisterType(func() confgenerator.MetricsReceiver { return &MetricsReceiverJetty{} })
 }
 
-type LoggingProcessorJettyAccess struct {
-	confgenerator.ConfigComponent `yaml:",inline"`
-}
+type LoggingProcessorMacroJettyAccess struct{}
 
-func (p LoggingProcessorJettyAccess) Components(ctx context.Context, tag string, uid string) []fluentbit.Component {
-	return genericAccessLogParser(ctx, p.Type(), tag, uid)
+func (p LoggingProcessorMacroJettyAccess) Expand(ctx context.Context) []confgenerator.InternalLoggingProcessor {
+	return genericAccessLogParser(ctx, p.Type())
 }
 
-func (LoggingProcessorJettyAccess) Type() string {
+func (LoggingProcessorMacroJettyAccess) Type() string {
 	return "jetty_access"
 }
 
 type LoggingReceiverJettyAccess struct {
```

Comment:
Please remove `LoggingReceiverJettyAccess` since it is not needed anymore after the refactor.